packages/hooks-react/src/useCheckAccess.ts

```diff
@@ -31,7 +31,7 @@ const useCheckAccess = () => {
         const hasAccess = await accountController.checkEntitlements(offerId);
 
         if (hasAccess) {
-          await accountController.reloadSubscriptions({ delay: 2000 }); // Delay needed for backend processing (Cleeng API returns empty subscription, even after accessGranted from entitlements call
+          await accountController.reloadSubscriptions({ retry: 10 });
```

Comment:
Should we keep the initial delay here? Do we ever see that the immediate response comes back populated already?

Reply:
I'm not entirely sure. I will test it and fall back to the initial delay of 2000 if needed.